jep/0000/README.adoc

```diff
@@ -0,0 +1,145 @@
+:toc: preamble
+:toclevels: 3
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+.Metadata
+[cols="1h,1"]
+|===
+| JEP
+| 0000
+
+| Title
+| Jenkins Twitter Policy for Contributors
+
+| Sponsor
+| link:https://github.com/tracymiranda[Tracy Miranda]
+
+| Status
+| Not Submitted :information_source:
+
+| Type
+| Process
+
+| Created
+| 2018-08-07
+
+| BDFL-Delegate
+| link:https://github.com/rtyler[R. Tyler Croy]
+
+|===
+
+== Abstract
+
+====
+This JEP describes the policy for Jenkins twitter accounts and covers:
+
+* Enabling a team of contributors
+* Jenkins Twitter policy
+* Regular reports on Twitter engagement
+
+====
+
+== Specification
+
+=== Enabling a Team of Contributors
+
+Tweetdeck's Team feature will be used to manage the team of contributors. 
+The following roles are recognized:
+
+* Owner - owner of the twitter account
+* Admin - can invite others to contribute
+* Contributor - creates content and can tweet, retweet and schedule tweets
+
+New contributors may be proposed at a governance meeting. 
+The Jenkins board will approve contributors using the following criteria:
+
+* Contributions - the candidate should actively be contributing to Jenkins community
+* Knowledge - the candidate should have indepth knowledge of Jenkins community and/or projects
+* Experience - the candidate should point to experience with Twitter/social media or submit sample tweets
+
+Once approved, the candidate should share their twitter account with the Twitter account owner
+The Twitter owner will invite the candidate to join the account Team via Tweetdeck. 
+
+New admins will typically be experienced contributors who the board approve as administrators. 
+
+==== Responsibilites of Contributors
+
+Admin and Contributors will make regular tweets or retweets to meet the goals below.
+Admin and Contributors agree to abide by the Jenkins Twitter policy. 
+The Twitter owner may revoke access if contributors violate this policy. 
+If contributors are inactive they may request to leave the team or the owner may revoke access. 
+
+=== Jenkins Twitter Policy
+
+==== Goals
+The goals of the @jenkinsci twitter account are:
+
+* Create awareness of Jenkins projects, releases, plugins and features
+* Promote awareness of Jenkins working groups, events, JAMs or talks about Jenkins
+* Build awareness of Jenkins and encourage contributions
+* Celebrate achievements of the community
+* Grow the open source community around Jenkins
+* Always be a trustworthy, reputable source of information
+
+==== Best Practices
+These are the best practices for contributors to follow:
+
+* Conduct - always abide by Jenkins community code of conduct .
+* Content - write or curate tweets that meet one of the goals above. 
+* Cadence - contributors should work together to maintain a regular cadence e.g. 3-5 tweets a day
```

Comment:
This seems like a _lot_. As in, tweeting pointless filler to reach a quota. Could we get 20-25 sample tweets for the last 7 day period, that would demonstrate that this is not unreasonably much?

Reply:
I think this sentence is easily fixed with "interesting/relevant tweets daily"

Reply:
Yes that works, will update in review pr